element-hq/element-x-android
# Remove duplicated code regarding user (room member and user profile) screens

## Content


Let `RoomMemberDetailsPresenter` delegates the computation of profile state to `UserProfilePresenter`, which allow to reduce code duplication.

## Motivation and context


Less code duplication (production and test).

Preparatory work for #3684

## Screenshots / GIFs



## Tests



There should be no change.
- Navigate to some room member details page
- Open matrix user profile, with member not in room (can be done using adb)

## Tested devices

- OS version(s):

## Checklist
